sdk/src/test/java/com/mapbox/maps/renderer/MapboxRenderThreadTest.kt

```diff
@@ -239,6 +241,57 @@ class MapboxRenderThreadTest {
     }
   }
 
+  @Test
+  fun needViewAnnotationSyncMapSynchronized() {
+    initRenderThread()
+    provideValidSurface()
+    // static method so does not matter which thread is called from -
+    // we need this interval for our render thread
+    ShadowChoreographer.setPostFrameCallbackDelay(16)
```

Comment:
is it possible that value for `setPostFrameCallbackDelay` and `idleHandler` can be different?
if yes, should you check it?

Reply:
no, moved to variable for better readability